src/OrchardCore.Modules/OrchardCore.ContentPreview/Views/ContentPreview.Button.cshtml

```diff
@@ -10,7 +10,7 @@
 
 @if (hasPreviewPermission)
 {
-    <a id="previewButton" target="_blank" class="btn btn-info" href="@Url.Action("Index", "Preview", new { area = "OrchardCore.ContentPreview", id = previewId })">@T["Preview"]</a>
+    <a id="previewButton" target="previewWindow" class="btn btn-info" href="@Url.Action("Index", "Preview", new { area = "OrchardCore.ContentPreview", id = previewId })">@T["Preview"]</a>
```

Comment:
@Skrypt do you agree with this change? I think I do (but equally if I have multiple Orchard instances running (dev and live), it will confuse me when it keeps swapping windows around on me?

Other changes look good to me.

Reply:
I'm not sure, to be honest. Could we not have a unique identifier for this target instead?

Reply:
add the host name as perfix?

Reply:
I would prefer removing it, and opending a seperate issue sowe can have a proper review of it.